### PR TITLE
[PKG-2080] ordered-set 4.1.0

### PR DESCRIPTION
**Pull request by skupr-anaconda**
License: https://github.com/rspeer/ordered-set/blob/release/4.1.0/MIT-LICENSE
Requirements:
- https://github.com/rspeer/ordered-set/blob/release/4.1.0/pyproject.toml
- https://github.com/rspeer/ordered-set/blob/release/4.1.0/setup.py

Actions:
1. Skip `py<37`
2. Add `--no-deps --no-build-isolation` to `script`
3. Add `wheel` to `host` and pin `flit-core` to `3.8.0`
4. Update `home` url
5. Add dev & doc urls
6. Add `license_family`
7. Add `description`

Notes:
- `nuitka 1.5.7` requires it https://github.com/AnacondaRecipes/nuitka-feedstock/pull/32

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,32 +10,37 @@ source:
...
   number: 0
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
+    - flit-core 3.8.0
     - pip
-    - python >=3.7
-    - flit-core
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml:21]: is this not needed anymore ? https://github.com/rspeer/ordered-set/blob/d921651b2737f5cfe711868fab558a8ef79e26ca/pyproject.toml#L2

Comment (skupr-anaconda) [reply recipe/meta.yaml:21]: It's here https://github.com/AnacondaRecipes/ordered-set-feedstock/pull/2/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR19

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by Jrice1317 — APPROVED** [review]
LGTM

**Pull request merged**